src/wandbot/ingestion/config.py

```diff
@@ -224,9 +241,8 @@ def _set_cache_paths(cls, values: "DataStoreConfig") -> "DataStoreConfig":
 
 class VectorStoreConfig(BaseSettings):
     name: str = "vectorstore"
-    embedding_dim: int = 1536
+    embeddings_model: str = "text-embedding-3-small"
```

Comment:
lets make this the large model? unless there is a dramatic increase in latency? but an extra few seconds is fine if it means a higher chance of better retrieval